webapp/src/index.tsx

```diff
@@ -429,6 +432,107 @@ export default class Plugin {
         registry.registerAdminConsoleCustomSetting('ICEHostPortOverride', ICEHostPortOverride);
         registry.registerAdminConsoleCustomSetting('ServerSideTURN', ServerSideTURN);
 
+        registry.registerSiteStatisticsHandler(async () => {
```

Comment:
Would it be cleaner to put this fn in its own file? I only ask because it seems a little out of place considering the conciseness of the other register calls above.

Reply:
Yes, I was planning to do it but then I couldn't think of a great place. I can add it to utils I suppose.